src/workspacer.Native/Native/WindowsManager.cs

```diff
@@ -14,7 +14,7 @@
 namespace workspacer
 {
     public delegate void WindowDelegate(IWindow window);
-    public delegate void WindowCreateDelegate(IWindow window, bool firstCreate);
+    public delegate void WindowAddDelegate(IWindow window, bool firstCreate);
```

Comment:
Changing the name of this delegate changes a published API. Do we need to do this?